python/source/flight.rst

```diff
@@ -0,0 +1,228 @@
+============
+Arrow Flight
+============
+
+Recipes related to leveraging Arrow Flight protocol
+
+.. contents::
+
+Using an Arrow Flight RPC server
```

Comment:
This title is a little generic for what it shows (I would have expected much less content/a much more minimal example), perhaps it could be called something like "A simple Parquet storage service with Arrow Flight"?

Reply:
Good suggestion, I struggled a bit with naming this recipe as it covers so many things as you pointed out.